content/community.json

```diff
@@ -78,6 +78,22 @@
       "date": "2021-07-14",
       "pictureUrl": "/img/community/events/dvc_learn.png"
     },
+    {
+      "url": "https://hasgeek.com/fifthelephant/mlops-conference/schedule",
+      "title": "HasGeek",
+      "description": "Milecia McGregor presents at HasGeek Conference from Fifth Elephant on Tuning Hyperparameters with DVC Experiments",
+      "city": "Virtual, Miami, FL",
+      "date": "2021-07-23",
+      "pictureUrl": "/img/community/events/hasgeek.png"
+    },
+    {
+      "url": "https://www.datasciencego.com/virtual",
```

Comment:
I think Jervis's talk shouldn't be here? Right?

Reply:
This was on a previous PR.  I don't know what happened, see below.